jekyll-dev.yml

```diff
@@ -229,3 +229,6 @@ jekyll-generator-single-source:
 mesh_helm_repo: kong-mesh/kong-mesh
 mesh_helm_install_name: kong-mesh
 mesh_disabled_versions: [dev]
+
+# binary options
+mesh_install_archive_name: kong
```

Comment:
It's `kong` and not `kong-mesh`?

Reply:
@slonka could you double-check this?

Reply:
> It's kong and not kong-mesh?

you're right, it's kong-mesh 
<img width="613" alt="image" src="https://github.com/Kong/docs.konghq.com/assets/2753650/96f591e1-bac8-4105-b881-3cb25435dac0">

let me change that

Reply:
```suggestion
mesh_install_archive_name: kong-mesh
```